adaptive/_version.py

```diff
@@ -1,208 +1,9 @@
-# This file is part of 'miniver': https://github.com/jbweston/miniver
-#
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Sorry, what?

Reply:
Oops, that must have been added by a detached pre-commit filter accidentally. Removed!